tests/ctst/features/quotas/Quotas.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Quota Management for APIs
+    This feature ensures that quotas are correctly set and honored
+    for different APIs.
+
+    @2.6.0
+    @PreMerge
+    @Quotas
+    @CronJob
+    @DataWrite
+    Scenario Outline: Quotas are evaluated during write operations
+        Given an action "<action>"
+        And an upload size of <uploadSize> B for the object ""
+        And a STORAGE_MANAGER type
```

Comment:
This sentence seems unclear, we might want to make more explanative on what is happening here

Reply:
I will not change it, it's used in depth in almost every test suite, not the objective of this PR to change all the tests